<launch>
    <arg name="map" default="/map"/>
    <arg name="is_costmap" default="false"/>
    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="(optenv ROS_ENV_LOADER)" user="$(arg user)" default="true"/>

	<node pkg="nav_goals_generator" type="generate_nav_goals.py" name="generate_nav_goals" output="screen">
        <param name="map_frame" value="$(arg map)"/>
        <param name="is_costmap" value="$(arg is_costmap)"/>
    </node>
</launch>
